src/Api/Controllers/ApiController.cs

```diff
@@ -7,6 +7,10 @@
 
 namespace Api.Controllers
 {
+    /// <summary>
+    /// Base class for AnnotationController
+    /// </summary>    
+    /// <response code="401">User is denied</response>
     [Produces("application/json")]
```

Comment:
![image](https://cloud.githubusercontent.com/assets/18174579/20651353/e65ac726-b4e3-11e6-8207-694b645437f7.png)
Doesn't look like that this in working in such a general way.